skale-nodes/ansible/run_monitor.sh

```diff
@@ -0,0 +1,10 @@
+cd ../..
+source venv/bin/activate
```

Comment:
It's better to remove activating virtualenv from script, because it's name and path depends on host and user.

Reply:
done